packages/lib-grommet-theme/src/index.js

```diff
@@ -1,21 +1,52 @@
 import deepFreeze from 'deep-freeze'
-import light from './light'
-import dark from './dark'
 
-const lightBlack = '#272727'
-const lightBrand = '#16979C'
-const darkGold = '#CC9200'
+const brand = '#00979d'
```

Comment:
Can we include a link to the style guide on InVision for reference?

Reply:
Yep, good idea